HISTORY.rst

```diff
@@ -7,6 +7,7 @@ Pending Release
 ---------------
 
 .. Insert new release notes below this line
+* Add security groups attribute.
```

Comment:
add a line before this to fix the markup complaint, it's not valid reStructuredText to leave this right next to the comment